src/placeholder.cls.php

```diff
@@ -250,7 +250,7 @@ private function _placeholder( $src, $size ) {
 		$tmp_placeholder = $this->_generate_placeholder_locally( $size );
 
 		// Store it to prepare for cron
-		$queue = $this->_load_queue( 'lqip' );
```

Comment:
The reason this didn't use `$this->_queue` like CCSS/UCSS is just it is historic code and didn't get the chance to refactor yet. After I approved this PR, if you are interested, you can consider to change it and drop the 2nd param of `save_queue()` func.